site/docs/components/input/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Input
+data:
+  description: Input provides an editable field in which users can enter text and numeric values. It is best used for short freeform content and data entry.
+  sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/blob/main/packages/core/src/input/Input.tsx"
+  package:
+    name: "@salt-ds/core"
+    initialVersion: "1.0.0"
+  alsoKnownAs: ["Text Field", "Text Box"]
+  relatedComponents: [{ name: "Multiline Input", relationship: "similarTo" }]
```

Comment:
Should form field be mentioned?

Reply:
I asked content in word doc if we can have something other than similar to and contains, for things like this where they can pair together, i.e. Input is a control of Form Field
cc @mikearildbrown ?

Reply:
Looks quite easy to add a new category in `site/src/layouts/DetailComponent/SecondarySidebar.tsx`. We can skip this for this PR for now. Not blocking issue.